### DISCUSSION TITLE
How to order monomials in a polynom when toString or toTex

### DISCUSSION THREAD

**Discussion by chrisbansart** [josdejong/mathjs#3000] [Q&A]
Hello,
I'm trying to order monomials in a standard way (highest degrees first) when I display polynomials.
Eg "3x+1+4x^2" should give "4x^2+3x+1"

Is there a specific option or rule I could apply to do that?

Thanks for your help
Chris

**Comment by josdejong** [comment]
You can probably use the function `simplify` for that. You can write a custom rule to apply this ordering. You'll have to write out the logic though to recognize the order of every part of the expression and re-order them when needed, that will require some effort.

Docs: https://mathjs.org/docs/reference/functions/simplify.html
